src/neha_Rathi/assignment15/AccountDetails.java

```diff
@@ -0,0 +1,171 @@
+/*Assignment - 15 : 9th April'2021
+
+A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+a) Person can debit or credit amount from respective bank.
+Hint : Switch case is expected here. [Please watch today's recording 
+
+again, in case of confusion.]
+
+b) A Method to show total balance [ICICI balance + HDFC balance + BOB 
+
+balance + SBI Balance]
+
+c) A Method to show total credit, debit operations
+output : Total credit operation - 7 times
+Total debit operation - 5 times
+
+d) A Method to show individual bank credit operations.
+output : ICICI bank Credit Operation - 2
+HDFC bank Credit Operation - 2
+SBI bank Credit Operation - 1
+BOB bank Credit Operation - 4
+
+e) A Method to show individual bank debit operations.
+output : ICICI bank Debit Operation - 2
+HDFC bank Debit Operation - 2
+SBI*/
+package neha_Rathi.assignment15;
+
+public class AccountDetails {
+	double totalBalanceIcici, totalBalanceHdfc, totalBalanceBob, totalBalanceSbi, totalBalance;
+	int debitCountIcici, debitCountHdfc, debitCountBob, debitCountSbi, creditCountIcici, creditCountHdfc,
+			creditCountBob, creditCountSbi, totalDebitCount, totalCreditCount;
+
+	void setBalance(double balace, String bankName) {
+
+		switch (bankName.toUpperCase()) {
+		case "ICICI":
+			totalBalanceIcici = balace;
+			break;
+		case "HDFC":
+			totalBalanceHdfc = balace;
+			break;
+		case "BOB":
+			totalBalanceBob = balace;
+			break;
+		case "SBI":
+			totalBalanceSbi = balace;
+			break;
+		default:
+			System.out.println("Invalid Bank");
+		}
+	}
+
+	void Operation(String op, String bankName, int amount) {
+		switch (op.toUpperCase()) {
+		default:
+			System.out.println("Invalid Transcation");
+		case "DEBIT":
+			switch (bankName.toUpperCase()) {
+
+			default:
+				System.out.println("Invalid bank");
+				break;
+			case "ICICI":
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.

Code snippet

int debit(int totalAmt, int debitAmt) {
		if (totalAmt < debitAmt) {
			return 0;
		} else
			return totalAmt - debitAmt;
}

totalAmt will your iciciAmt (instance variable)